_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -3115,9 +3131,13 @@ mfxStatus VideoVPPHW::VppFrameCheck(
 #endif
     MFX_CHECK_STS(sts);
 
+    bool useCopyPassThrough =
+        (m_config.m_bCopyPassThroughEnable && !IsRoiDifferent(pTask->input.pSurf, pTask->output.pSurf))?
```

Comment:
Why you don't want to move this condition to UseCopyPassThrough() as well?

Reply:
Done.